azure.gemspec

```diff
@@ -35,8 +35,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('azure-core',              '~> 0.1')
   s.add_runtime_dependency('faraday',                 '~> 0.9')
   s.add_runtime_dependency('faraday_middleware',      '~> 0.10')
-  s.add_runtime_dependency('json',                    '~> 1.8')
-  s.add_runtime_dependency('mime-types',              ['>= 1', '< 3.0'])  # vagrant-share and other stuff relies on 1
+  s.add_runtime_dependency('mime-types',              ['>= 1', '< 4.0'])  # vagrant-share and other stuff relies on 1
```

Comment:
removing 4.0 will result in a warning as open ended dependency. And the latest version is 3.1